src/bosh-director/lib/bosh/director/addon/addon_filter.rb

```diff
@@ -1,5 +1,5 @@
 module Bosh::Director
-  module RuntimeConfig
+  module Addon
     class AddonFilter
```

Comment:
Can we rename this class to `Filter` since it's now namespaced in `Addon` to avoid redundancy?